src/common/apollo/utils.ts

```diff
@@ -2,6 +2,16 @@ import { ApolloClient, NormalizedCacheObject } from "@apollo/client";
 
 import { Connection, MenuItem } from "../../types";
 
+export const excludeArgs =
+  (excludedArgs: string[]) =>
+  (
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    args: Record<string, any> | null
```

Comment:
I assume unknown didn't pass type checking here?

Reply:
Hmm need to check. This was actually just copy paste of what I had done in the other project

Reply:
changed any to unknown